tests/test_strings.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from core_lib.helpers.string import snake_to_camel, camel_to_snake
+
+
+class TestStrings(unittest.TestCase):
+
+    def test_snake_to_camel(self):
+        self.assertNotEqual(snake_to_camel("hello_world"), None)
+        self.assertEqual(snake_to_camel("hello_world"), "HelloWorld")
+        self.assertNotEqual(snake_to_camel("this_is_snake_to_camel"), None)

```

Comment:
is we using the same input for two test cases?? 
we can make only a single assertion